lib/human_player.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class HumanPlayer
+  attr_reader :token
+
+  def initialize(token:, user_input:)
+    @token = token
+    @user_input = user_input
+  end
+
+  def position(board:)
```

Comment:
it seems weird to me that we have to expose board to player. I would imagine, the player gives an input, the game takes it in, validate it and tells the board to register the move.